### ISSUE TITLE
Docs?

### ISSUE THREAD

**Issue by adamdonahue** [Glympse/terraform-provider-nifi#24]
Is there documentation on the _resources_ exposed by this provider, other than by trolling through the examples or source code?  "Syntax" refers only to how to connect to the provider.

**Comment by zaquestion** [comment]
Not to my knowledge I'm afraid, I can peak around our internal usage usage and see if I can provide a realworld example. Other than that @egorpushkin is the only other person I'm aware that might be able to help.

**Comment by adamdonahue** [comment]
Looks like it maps more-or-less to the API, so the Nifi REST API
documentation fills the gap. Thanks.

**Comment by egorpushkin** [comment]
@adamdonahue Correct. The goal was to mimic NiFi API to make the use of this provider intuitive for those familiar with the original API and NiFi concepts in general. That said, it is always nice to have docs available in-place without having to toggle between resources. Please, feel free to submit PR with documentation covering other resources.
